tidb-retool-integration.md

```diff
@@ -0,0 +1,22 @@
+# TiDB and Retool Integration
```

Comment:
If this document is only applicable to TiDB Cloud, please place the file in the `tidb-cloud` directory.